src/main/java/seedu/address/storage/JsonAdaptedClient.java

```diff
@@ -21,7 +21,7 @@
  * Jackson-friendly version of {@link Client}.
  */
 class JsonAdaptedClient {
-
+    // TODO: add procedures
```

Comment:
Apart from adding the procedure as a variable here, could it also be better to update the other classes and methods?

For example:
``
JsonAdaptedClient(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
            @JsonProperty("email") String email, @JsonProperty("address") String address,
            @JsonProperty("tagged") List<JsonAdaptedTag> tagged, @JsonProperty("procedure") List<JsonAdaptedProcedure> procedure) {
``

or updating `public JsonAdaptedClient(Client source)` to include the procedures as well?